<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtr_Gaussa</class>
 <widget class="QWidget" name="Filtr_Gaussa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="lButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>116</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pOk">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pAnuluj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Cofnij</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="wPromienia">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>56</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
